exec/java-exec/src/main/java/org/apache/drill/exec/coord/zk/ZookeeperClient.java

```diff
@@ -133,13 +137,50 @@ public boolean hasPath(final String path, final boolean consistent) {
    * the check is eventually consistent.
    *
    * @param path  target path
+   * @param consistent consistency flag
```

Comment:
Diff didn't show the whole comment which is:
If the flag consistent is set, the check is consistent as it is made against Zookeeper directly. Otherwise,
the check is eventually consistent.